lib/Dialect/TritonGPU/Transforms/Pipeliner/MMAv5PipelineUtility.cpp

```diff
@@ -22,7 +22,7 @@ bool ttng::MMAv5PipelineableOperandsHelper::comesFromLoadOrOutsideLoop(
   if (!v.getDefiningOp()) {
     return false;
   }
-  while (isa<ttg::MemDescTransOp, ttg::MemDescReshapeOp>(v.getDefiningOp())) {
+  while (v.getDefiningOp()->hasTrait<OpTrait::MemDescViewTrait>()) {
```

Comment:
actually I'm not sure we want to skip subview in this case?

Reply:
I haven't seen this be a problem in practice yet, but one of the patterns I have seen is

```mlir
%buffer = ttg.local_alloc : () -> memdesc<1x128x64xf16,
%buffer0 = ttg.memdesc_subview %buffer[%c0_i32, %c0_i32, %c0_i32]
scf.for ...
   tcgen_5_mma %buffer0, %rhs, %acc
```

This can end up happening just because various utilities don't bother peepholing 

```mlir
%buffer = ttg.local_alloc : () -> memdesc<1x128x64xf16,
%buffer0 = ttg.memdesc_subview %buffer[%c0_i32, %c0_i32, %c0_i32]
```

into

```mlir
%buffer0 = ttg.local_alloc : () -> memdesc<128x64xf16,
```

I think we still want the tcgen_5_mma in this case to be software pipelined if it can be.

Reply:
won't this happens though:
```
%buffer = ttg.local_alloc : () -> memdesc<3x128x64xf16,
scf.for ... {
  %buffer0 = ttg.memdesc_subview %buffer[%iv, %c0_i32, %c0_i32]
  tcgen_5_mma %buffer0, %rhs, %acc
```
in this case we don't want to return true?

Reply:
Hmm, I guess I'm not sure what happens if the accumulator is already multibuffered. I can revert this change until there is a concrete use case